etc/pylintrc

```diff
@@ -1,6 +1,8 @@
 [MASTER]
 extension-pkg-whitelist=numpy,scipy
+ignore=conf.py
```

Comment:
I'm not sure if this is working correctly, as when I tried running this i.e. `improver tests pylint > temp.txt`, it seemed like conf.py had been analysed by pylint. Perhaps this needs to be `/etc/conf.py`